.github/workflows/push_hypernext_image.yaml

```diff
@@ -0,0 +1,51 @@
+name: Deploy images of hypernext to Docker
+
+on:
+    push:
+        branches:
+          - hypernext
+    # run every day at 1:00
+    schedule:
+    - cron: "00 01 * * *"
+
+jobs:
+    buildandpush:
+        runs-on: ubuntu-latest
+        env:
+            BRANCHNAME: hypernext
+            DOCKER_FILE: Dockerfile
+        steps:    
+            - name: Checkout elabimg repo on latest tag
+              uses: actions/checkout@v2
+              with:
+                  repository: "elabftw/elabimg"
+                  ref: ${{ env.BRANCHNAME }}
+                  
+            # https://github.com/docker/setup-qemu-action#usage
+            - name: Set up QEMU
+              uses: docker/setup-qemu-action@v1
+
+            # https://github.com/marketplace/actions/docker-setup-buildx
+            - name: Set up Docker Buildx
+              uses: docker/setup-buildx-action@v1
+
+            # https://github.com/docker/login-action#docker-hub
+            - name: Login to Docker Hub
+              uses: docker/login-action@v1
+              with:
+                username: ${{ secrets.DOCKER_HUB_USERNAME }}
+                password: ${{ secrets.DOCKER_HUB_TOKEN }}
+
+            # https://github.com/docker/build-push-action#multi-platform-image
+            - name: Build latest release for all platforms and push to Docker Hub
+              uses: docker/build-push-action@v2
+              with:
+                context: .
+                file: ${{ env.DOCKER_FILE }}
+                platforms: linux/arm64,linux/amd64
+                push: true
+                build-args: |
+                    ELABFTW_VERSION=${{ env.BRANCHNAME }}
+                    ELABIMG_VERSION=${{ env.BRANCHNAME }}
+                tags: |
+                    ${{ secrets.DOCKER_HUB_USERNAME }}/${{ secrets.DOCKER_HUB_REPOSITORY }}:${{ env.BRANCHNAME }}
```

Comment:
I think this should be changed. The name of the image should be `elabftw/elabimg:<branch>`

No need to have the hub repository as a secret, too. So it can be hardcoded except for the branch that will change.

Reply:
done